tracer/src/Datadog.Trace/Tagging/TraceTagCollection.cs

```diff
@@ -7,254 +7,136 @@
 
 using System;
 using System.Collections.Generic;
-using System.Threading;
-using Datadog.Trace.Util;
+using Datadog.Trace.Logging;
 
-namespace Datadog.Trace.Tagging
+namespace Datadog.Trace.Tagging;
```

Comment:
While I'm all in on file-scoped namespaces, I'm very against making that change at the same time as other changes as it messes with git diff something awful 😉